export-ingester/export_ingester/sftp.py

```diff
@@ -7,9 +7,18 @@
 from .models import ExportList, ExportModel
 
 
-class SFTPClient:
-    def __init__(self, client: asyncssh.SFTPClient):
-        self.client = client
+class SSHClient:
+    def __init__(self, conn: asyncssh.SSHClientConnection):
+        self.conn = conn
+
+    async def remote_sbatch(self, sbatch_command: str, remote: str) -> str:
+        """Executes the sbatch command on the remote server."""
+        try:
+            result = await self.conn.run(f"{sbatch_command} {remote}", check=True)
+            return result.stdout  # Return the output of the sbatch command
+        except asyncssh.Error as e:
+            print(f"Error running sbatch command: {str(e)}")
```

Comment:
It's better to log rather than print. It's a bit unclear why an empty string is returned. Would it not be better to raise your own error?

Reply:
Implemented some logging. Maybe I could also raise my own error. What do you think?